src/Cache/RedisCacheTagsChecksum.php

```diff
@@ -74,6 +62,9 @@ public function invalidateTags(array $tags) {
    * {@inheritdoc}
    */
   public function getCurrentChecksum(array $tags) {
+    /*
+     * @todo Restore cache
```

Comment:
As said here, this is still a proof of concept, but it do work nicely. The cache may be restored in order to gain some round trips to Redis, I'll do this, I am thought making the pull request as-is to have your feedback faster.